model-optimizer/unit_tests/extensions/back/TransposeDFT_test.py

```diff
@@ -0,0 +1,185 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import unittest
+
+from extensions.back.TransposeDFT import TransposeDFT
+from mo.front.common.partial_infer.utils import int64_array
+from mo.utils.ir_engine.compare_graphs import compare_graphs
+from unit_tests.utils.graph import build_graph
+
+
+dft_graph_node_attrs = {
```

Comment:
Consider using helper functions to build test graph as in the model-optimizer/unit_tests/extensions/back/InterpolateReshape_test.py. This will allow to dramatically reduce the code size. The same comment for other middle/back phase transformations tests

Reply:
Done.